helloworld/greet/greeting.py

```diff
@@ -2,29 +2,20 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 import random
+from typing import List
 
 from helloworld.util.lang import LanguageTranslator
-from helloworld.util.resources import get_lines
 
 
 class Greeter:
-    def __init__(self):
-        self._greeting_selector = GreetingSelector()
-        self._language_translator = LanguageTranslator()
+    def __init__(self, greetings: List[str], languages: List[str]) -> None:
+        self._greetings = greetings
+        self._language_translator = LanguageTranslator(languages)
 
     def translated_greeting(self) -> str:
-        return self._language_translator.translate_to_random_language(
-            self._greeting_selector.pick_greeting()
-        )
+        random_greeting = random.choice(self._greetings)
+        return self._language_translator.translate_to_random_language(random_greeting)
 
     def greet(self, name: str) -> str:
         greeting = self.translated_greeting()
         return f"{greeting}, {name}!"
-
-
-class GreetingSelector:
-    def __init__(self):
-        self._greetings = get_lines(__name__, "greetings.txt")
```

Comment:
See https://github.com/pantsbuild/example-python/pull/13. I argue that it makes things simpler/more traditional Python to have `greet.py` and `lang.py` take these values as normal parameters, rather than reading a hardcoded file. Separation of concerns.